.gitignore

```diff
@@ -2,3 +2,5 @@
 target/
 **/*.rs.bk
 .DS_Store
+exercises/clippy/Cargo.toml
```

Comment:
I'm assuming the separate Cargo.toml is so that Clippy doesn't lint the entire project?

Reply:
Yes, I was hoping that there is an option to invoke Clippy without Cargo and without a Cargo.toml and just pass it the bin with an option, but it seems that this is not possible.
This is why I'm generating a custom Cargo.toml just to invoke Clippy with it